packages/hackathon/artifact.ipynb

```diff
@@ -2,56 +2,119 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA = {\n",
-    "    'client_id':'',\n",
-    "    'client_secret':''\n",
-    "}\n",
+    "import looker_sdk\n",
+    "from looker_sdk import api_settings\n",
+    "\n",
+    "class MyApiSettings(api_settings.ApiSettings):\n",
+    "    def __init__(self, *args, **kw_args):\n",
+    "        super().__init__(*args, **kw_args)\n",
+    "\n",
+    "    def read_config(self) -> api_settings.SettingsConfig:\n",
+    "        config = super().read_config()\n",
+    "        config[\"client_id\"] = \"BxCqx9Wbrn2pqvCggsvj\"\n",
+    "        config[\"client_secret\"] = \"zqsSb7NXBRz9DvkZxNwrDfCC\"\n",
+    "        config[\"base_url\"] = \"https://hack.looker.com:19999\"\n",
     "\n",
-    "BASE_URL = 'http://localhost:19999'"
+    "        return config\n",
+    "\n",
+    "sdk = looker_sdk.init40(config_settings=MyApiSettings())"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Login\n",
-    "import requests \n",
+    "import json \n",
+    "import csv\n",
     "\n",
-    "response = requests.post(BASE_URL + '/api/4.0/login',data=DATA)\n",
-    "print(response.status_code)\n",
-    "access_token = response.json()['access_token']\n",
-    "headers= {\n",
-    "    'Authorization': 'token ' + access_token,\n",
-    "    'x-looker-appid': 'marketplace_extension_hackathon::hackathon'\n",
-    "}\n",
-    "response = requests.get(\n",
-    "    BASE_URL + '/api/4.0/artifact/namespaces',\n",
-    "    headers=headers)\n",
-    "\n",
-    "print(response.json())"
+    "def getRows(tableName):\n",
+    "    artifacts = sdk.search_artifacts(key=tableName+'%', namespace='Hackathon')\n",
+    "    rows = []\n",
+    "    for a in artifacts:\n",
+    "        row = json.loads(a.value)\n",
+    "        row['key'] = a.key\n",
+    "        rows.append(row)\n",
+    "    return rows\n",
+    "\n",
+    "def toCSV(rows, fileName):\n",
```

Comment:
Is this a temporary utility? By adding more Python functionality we're not providing functionality that could be in the Hackathon app admin section.

Reply:
Faster for you, perhaps. However, it's easy enough to replace later.

Reply:
Also a lot of this functionality will be hopefully explores/dashboards. Shouldn't need to be in our hack app.